src/clap_app.rs

```diff
@@ -304,6 +304,17 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                 .hidden(true)
                 .help("Show path to the configuration file."),
         )
+        .arg(
+            Arg::with_name("italic-text")
+                .long("italic-text")
+                .takes_value(true)
+                .multiple(false)
+                .value_name("always|*never*")
+                .possible_values(&["always", "never"])
+                .default_value("never")
+                .hide_default_value(true)
+                .help("Use italics in output (always, *never*)"),
```

Comment:
Can we also add a `long_help(..)` here:

"Specify when to use ANSI sequences for italic text in the output. Possible values: always, *never*."